modules/openapi-generator/src/test/resources/3_0/python-experimental/petstore-with-fake-endpoints-models-for-testing-with-http-signature.yaml

```diff
@@ -1202,6 +1203,7 @@ components:
         shipDate:
           type: string
           format: date-time
+          example: '2020-02-02T20:20:20.000222Z'
```

Comment:
I suggest adding a few unit tests to show deserialization with time zone. Since `shipDate` is part or the `Order` schema, you can edit samples/openapi3/client/petstore/python-experimental/test/test_order.py to add unit tests there.
Currently there are no unit tests in that particular file, you can look for unit test examples in test_drawing.py and samples/openapi3/client/petstore/python-experimental/tests/*.py.

Reply:
I have added unit test to check `toDefaultValue` method.